server/game/cards/04.2-NTB/Consecration.js

```diff
@@ -0,0 +1,39 @@
+const SpellCard = require('../../spellcard.js');
+
+class Consecration extends SpellCard {
+    setupCardAbilities() {
+        this.spellAction({
+            title: 'Consecration',
+            playType: 'cheatin resolution',
+            target: {
+                activePromptTitle: 'Select your dude to Consecrate',
+                waitingPromptTitle: 'Waiting for opponent to select a dude',
+                cardCondition: { location: 'play area', controller: 'current'},
+                cardType: ['dude']
+            },
+            difficulty: 7,
+            actionContext: { card: this, gameAction: ['increaseBullets', 'increaseInfluence', 'setAsStud'] },
```

Comment:
This will cause that if the parent cannot be affected even by one of the effects, all will be prevented. Is that ok or should be only the one effect prevented?
For example if blessed cannot have bullets increased, not only the bullet increase will be prevented, but also influence and stud effect.